pegasus/sites.v3/code.org/public/learn/legacy.haml

```diff
@@ -1,24 +0,0 @@
----
-title: Learn
-theme: responsive
-social:
-  "og:description": "Anyone can learn the basic concepts of Computer Science. Choose an intro activity to start learning."
-  "og:image": "http://code.org/images/fit-520/codehoc3.jpg"
-  "og:image:width": 520
-  "og:image:height": 391
-  "og:video": ""
-  "og:video:width": ""
-  "og:video:height": ""
-  "og:video:type": ""
----
-
-%link{type: "text/css", href: "/css/learn_tabs.css", rel: "stylesheet"}
-%link{type: "text/css", href: "/shared/css/course-blocks.css", rel: "stylesheet"}
-%link{type: "text/css", href: "/css/learn-carousel.css", rel: "stylesheet"}
-%script{type: "text/javascript", src: "/js/bootstrap.min.js"}
-
-= view 'mobilecheck.js'
-
-= view :learn_teacher_banner
```

Comment:
Looks like this is the only file exclusively used by this file, so good job catching that it's okay for deletion.